api/python/provisioner/errors.py

```diff
@@ -16,9 +16,37 @@
 #
 
 import subprocess
+import json
 from typing import Dict, Union, Any
 
 
+class CustomFormat():
+
+    def __init__(self, obj):
+        self._obj = obj
+
+    def dict_to_str(self):
+
+        if isinstance(self._obj, dict):
+            res = ''
+
+            for key, value in self._obj.items():
+                if isinstance(value, dict):
+                    value = json.dumps(value, indent=4)
+                res = res + f"{key}: {value},\n"
```

Comment:
I guess there would be no indentation for wrapped parts of a record, like that:
```
record1 ....
<json data starts for the line beginning>
record2 ...
```

Reply:
I have given indentation for key and value but in case of value, it will be displayed as it is if they do not have tab after newline then result will be displayed it as it is.